### DISCUSSION TITLE
Question.

### DISCUSSION THREAD

**Discussion by EvgenLab** [Microgenital/Giants_Software_USB_Beacon#5] [General]
Is there any way to make a homemade arduino lighthouse to work with the game farming simulator 22?

**Comment by Microgenital** [comment]
I am not familiar with Arduino, but in principle, it should be possible for it to present itself as the GiantsBeacon and respond to the game's command. However, I don’t know how this works or what is needed for it. You might need to ask in an Arduino forum or something similar.
Edit: I'll move it to the discussions section, as it doesn't represent an error in the code.
